DataFormats/TrackerRecHit2D/src/VectorHit.cc

```diff
@@ -214,8 +214,8 @@ std::pair<double, double> VectorHit::curvatureORphi(std::string curvORphi) const
                 pow(gPositionLower.y(), 2) * gPositionUpper.y() - gPositionLower.y() * pow(gPositionUpper.y(), 2);
 
     //radius of circle
-    double rho = sqrt(r12 * r22 * h3) / (2. * h1);
-    curvature = 1. / rho;
+    double invRho2 = (2. * h1) / (r12 * r22 * h3);
```

Comment:
You fixed this according an initial comment of mine that I was not fast enough to remove: sorry for that!
In fact, in the original formulation, the square root is only made with the numerator.
Perhaps you can make use the of the "denom1" you are defining below:
```C++
    double rho = 1./ denom1 / (2. * h1);
```

Reply:
Or, alternatively:
```C++
    double invRho2 = (4. * h1 * h1) / (r12 * r22 * h3);
```

Reply:
Thanks for double checking, I overlooked that. I have fixed it according to the second suggestion.